frontend/js/client.js

```diff
@@ -266,6 +271,57 @@ function handleServerInfo(config) {
     redirectURL = config.redirectURL;
 }
 
+async function refreshCodecAndBitrate() {
+    try {
+        if (!thereIsPeerConnections()) return;
+        for (let peerId in peerConnections) {
+            let videoSelectCodecChanged = false;
+            let videoSenderMaxBitrateChanged = false;
+            let videoTransceiver = peerConnections[peerId]
+                .getTransceivers()
+                .find((s) => s.sender.track && s.sender.track.kind === 'video');
+            if (videoTransceiver) {
+                // Get the sender's parameters
+                const videoSender = videoTransceiver.sender;
+                const videoParameters = await videoSender.getParameters();
+                // Update codec in real time
+                if (config.videoSenderCodec !== 'default') {
+                    const senderCapabilities = RTCRtpSender.getCapabilities(videoSender.track.kind);
+                    const selectedCodec = senderCapabilities.codecs.find((codec) =>
```

Comment:
I am not sure if Array.find is enough. Codec may have other different property (like profiles). In the worst case, you might select a codec whose profile is not supported on the receiver (but the receiver may support other profiles).